scripts/timer.py

```diff
@@ -0,0 +1,73 @@
+import json
```

Comment:
Change this file name to chronic_txn_sync.py
may be you can think a better name?